modules/xfem/include/userobjects/GeometricCutUserObject.h

```diff
@@ -15,10 +15,21 @@
 #include "libmesh/libmesh.h" // libMesh::invalid_uint
 #include "libmesh/elem.h"
 
+#include <array>
+
 using namespace libMesh;
 
+struct CutPoint
```

Comment:
This `CutPoint` struct seems really specific to your mesh-based cutter, so it would be better to put it in the header of that class.